examples/role-resources/role.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 rules:
 # Permissions for every EventListener deployment to function
 - apiGroups: ["tekton.dev"]
-  resources: ["eventlisteners", "triggerbindings", "triggertemplates", "tasks", "taskruns"]
+  resources: ["clustertriggerbindings", "eventlisteners", "triggerbindings", "triggertemplates", "tasks", "taskruns"]
```

Comment:
Wouldn't this have to be a cluster role since cluster trigger bindings are cluster scoped resources?

It would be nice to keep the existing role resources to use the existing role binding while adding a new one for the cluster trigger binding examples

Reply:
I have fixed this, with new-files and structure